AmplifyPlugins/API/Sources/AWSAPIPlugin/AWSAPIPlugin+Resettable.swift

```diff
@@ -27,9 +26,6 @@ extension AWSAPIPlugin: Resettable {
         }
 
         subscriptionConnectionFactory = nil
-
-        // Issue: https://github.com/aws-amplify/amplify-ios/issues/2120
-        // AwsCommonRuntimeKit.cleanUp()        
```

Comment:
are we saying that the issue should be resolved and we should not be calling `AwsCommonRuntimeKit.cleanUp()` ?

Reply:
opened an issue with crt team : https://github.com/awslabs/aws-crt-swift/issues/134

Reply:
This is an optional feature as confirmed by the CRT team and is expected to hang if there is reference to CRT resource.